src/bundle/Resources/public/scss/_alerts.scss

```diff
@@ -200,6 +325,7 @@
         }
     }
 
+    //Remove in 5.0
```

Comment:
maybe add @depracated , simpler to find it / ensure all depracations are removed in 5.0?

Reply:
I can add Deprecated but right now I can see only 2 places with deprecated label